onnxruntime/core/providers/nuphar/scripts/symbolic_shape_infer.py

```diff
@@ -240,7 +274,34 @@ def _onnx_infer_single_node(self, node):
             vi.CopyFrom(self.tmp_mp_.graph.output[i_o])
             self.known_vi_[vi.name] = vi
 
-    def _get_int_values(self, node):
+    def _onnx_infer_subgraph(self, node, subgraph):
+        if self.verbose_ > 2:
```

Comment:
I don't think this will handle an If subgraph (input length to If is 1, but subgraph has 0 explicit inputs) so name may be misleading if someone tried to add If processing in the future. It may work if you remove the assert about the input length for the node matching the subgraph.